sys/include/net/netdev2_test.h

```diff
@@ -0,0 +1,306 @@
+/*
```

Comment:
No. It is not part of it. I experimented on lwIP and emb6 with this framework just fine :-)